src/game/client/components/chat.cpp

```diff
@@ -87,6 +88,22 @@ void CChat::OnReset()
 	}
 }
 
+void CChat::OnMapLoad()
+{
+	if(Client()->State() == IClient::STATE_LOADING)
+	{
+		if(m_FirstMap)
+			m_FirstMap = false;
+		else
+		{
+			// display map rotation marker in chat
+			char aBuf[256];
+			str_format(aBuf, sizeof(aBuf), Localize("Map changed to %s"), Client()->GetCurrentMapName());
```

Comment:
Maybe put '' around the map name? Like it's done with names.

Reply:
:+1: